st2common/st2common/models/db/stormbase.py

```diff
@@ -110,11 +111,11 @@ class StormBaseDB(StormFoundationDB):
 class EscapedDictField(me.DictField):
 
     def to_mongo(self, value):
-        value = mongoescape.escape_chars(value)
+        value = mongoescape.escape_chars(copy.deepcopy(value))
         return super(EscapedDictField, self).to_mongo(value)
 
     def to_python(self, value):
-        value = super(EscapedDictField, self).to_python(value)
+        value = super(EscapedDictField, self).to_python(copy.deepcopy(value))
```

Comment:
I did it to be symmetric with to_mongo. Maybe leave it alone until a bug is identified?